WordPress/Classes/ViewRelated/Jetpack/Install/JetpackInstallPluginHelper.swift

```diff
@@ -125,7 +125,11 @@ class JetpackInstallPluginHelper: NSObject {
 private extension JetpackInstallPluginHelper {
 
     static var isFeatureEnabled: Bool {
-        FeatureFlag.jetpackIndividualPluginSupport.enabled || FeatureFlag.wordPressIndividualPluginSupport.enabled
```

Comment:
Previously, this would be a risk of logic error (when this is disabled in Jetpack, but the WP's feature flag is forcefully enabled from remote).